Assets/MixedRealityToolkit/Utilities/Lines/Renderers/StripMeshLineRenderer.cs

```diff
@@ -5,26 +5,30 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.Core.Utilities.Lines.Renderers
+namespace Microsoft.MixedReality.Toolkit.Core.Utilities.Lines
 {
     /// <summary>
     /// Draws a strip of polygons along the line
     /// </summary>
-    public class StripMeshLineRenderer : BaseMixedRealityLineRenderer
+    public class StripMeshLineRenderer : BaseLineRenderer
     {
         [Header("Strip Mesh Settings")]
 
         [SerializeField]
         private Material lineMaterial = null;
-
         [SerializeField]
         private float uvOffset = 0f;
 
-        private Mesh stripMesh;
-        private Material lineMatInstance;
+        [SerializeField]
+        [HideInInspector]
         private MeshRenderer stripMeshRenderer;
```

Comment:
Should we (do we, and I missed it?) try to grab this [in `OnValidate`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/b170096ebfb6ed9a2c33191df0f500e8f11c5876/Assets/MixedRealityToolkit-SDK/Features/Audio/Influencers/AudioLoFiEffect.cs#L96) so it's serialized at edit time?

Reply:
We create / destroy this object in OnEnable / Disable, and the component runs in edit mode, so it will be created and serialized at edit time by default. (Does that answer your question?)

Reply:
Didn't realize the component runs in edit mode! Great!